wagtail/admin/templatetags/wagtailuserbar.py

```diff
@@ -70,8 +68,6 @@ def wagtailuserbar(context, position="bottom-right"):
                     AdminItem(),
                     ExplorePageItem(revision.content_object),
                     EditPageItem(revision.content_object),
-                    ApproveModerationEditPageItem(revision),
-                    RejectModerationEditPageItem(revision),
```

Comment:
We should update https://github.com/wagtail/wagtail/blob/cba75351a8b9512f5fa2b89408879f59f7d064cd/docs/reference/hooks.md?plain=1#L865 to not mention "moderation"